### ISSUE TITLE
Is not possible import Spring Security

### ISSUE THREAD

**Issue by manueljordan** [eclipse/buildship#1061]
Some days ago in the same machine I was able to import the `Spring Security` project based on Gradle on the `STS 4.9.0` without any problems.

BTW Spring Security has a gradle wrapper: `gradle/wrapper/gradle-wrapper.properties`

```
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
```

But since yesterday happens the following for a new directory and new workspace for this project.

When I import does not import as is expected, it has the following:

![01](https://user-images.githubusercontent.com/2552795/107123221-5bbf0700-686a-11eb-9fbf-210346f48e96.png)

The project is imported but it does not show the sub projects as before:

![02](https://user-images.githubusercontent.com/2552795/107123235-6da0aa00-686a-11eb-9e81-377ac951da6b.png)

If in the root directory I do right click and _refresh_ and then: 

* `Right Click` -> `Gradle` -> `Refresh Gradle Project`

arises:

![03](https://user-images.githubusercontent.com/2552795/107123528-fbc96000-686b-11eb-8f83-83a1ec60905f.png)

![04](https://user-images.githubusercontent.com/2552795/107123534-071c8b80-686c-11eb-9ee4-9bfc811f1213.png)

![05](https://user-images.githubusercontent.com/2552795/107123536-0d126c80-686c-11eb-9be3-10d8c53226b0.png)
 
More details if I do double clic

![06](https://user-images.githubusercontent.com/2552795/107123590-55ca2580-686c-11eb-89ca-7d6f90d9ba0b.png)

![07](https://user-images.githubusercontent.com/2552795/107123594-5c589d00-686c-11eb-9eff-a172204b91bd.png)

Not sure what happened, I don't remember that I did do an upgrade for the BuildShip plugin

**Comment by donat** [comment]
I've downloaded STS 4.9.0, used the same JDK and import settings, but still, it works for me. 

The easiest way to update Buildship is via the marketplace. Just search for `Buildship` and click the button with the label `installed`: it will offer you the update.

**Comment by manueljordan** [comment]
Hi, yes, I work through Eclipse Marketplace and even it is marked how a favourite, and I have the latest release:

![01](https://user-images.githubusercontent.com/2552795/107219850-a57d2e00-69df-11eb-82c7-e0516bf534d1.png)

Below the explicit version.

![02](https://user-images.githubusercontent.com/2552795/107219902-b3cb4a00-69df-11eb-9d44-4219b8f639c9.png)

If you can share some instructions to enable logging in debug mode, perhaps more info I would share

**Comment by manueljordan** [comment]
I found the solution, and it was about the final scenario to check.

I installed the `spring-javaformat` plugin (it never was installed on Windows - it is the unique difference). 
Since its installation I had problems. More details here:

* [Problems to import and compile the spring-javaformat project #259](https://github.com/spring-io/spring-javaformat/issues/259)

I uninstalled the plugin, now all is how is expected

![Screen Shot 2021-02-17 at 12 25 10 PM](https://user-images.githubusercontent.com/2552795/108243063-a74f9b80-711b-11eb-83f5-5a0c2989bacd.png)

Really wondered why the `spring-javaformat` plugin had affected indirectly to the Buildship plugin and the STS itself.
